app/javascript/components/admin/UserModal.js

```diff
@@ -122,11 +123,11 @@ class UserModal extends React.Component {
                 inputId="role-select"
                 name="role"
                 value={
-                  this.props.initialUserData.role_title
-                    ? { label: this.props.initialUserData.role_title, value: this.props.initialUserData.jurisdiction_path }
-                    : { label: this.props.roles[0], value: this.props.roles[0] }
+                  this.state.roleTitle
+                    ? { label: this.state.roleTitle, value: this.state.roleTitle }
+                    : { label: this.state.roles[0], value: this.state.roles[0] }
```

Comment:
you could keep this `this.props.roles`, see above comment

Reply:
Same commit. :-) [ea9253d6956658ca2db9763a9140c370e806cb31](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/ea9253d6956658ca2db9763a9140c370e806cb31)